File: Sem4/MPP_Templates/ZboruriCSharp/model/Bilet.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace model
{
    [Serializable]
    public class Bilet : HasId<int>
    {
        public int Id { get; set; }
        public int ZborId { get; set; }
        public int ClientId { get; set; }
        public String Turisti { get; set; }
        public int Locuri { get; set; }

        public Bilet(int Id, int ZborId, int ClientId, String Turisti, int Locuri)
        {
            this.Id = Id;
            this.ZborId = ZborId;
            this.ClientId = ClientId;
            this.Turisti = Turisti;
            this.Locuri = Locuri;
        }

        public override String ToString()
        {
            return "Id=" + Id + "|ZborId=" + ZborId + "|ClientId=" + ClientId + "|Turisti=" + Turisti + "|Locuri=" + Locuri;
        }
    }
}

File: Sem4/MPP_Templates/Zboruri/FlightModel/src/main/java/flight/domain/Bilet.java
package flight.domain;

import java.io.Serializable;

public class Bilet implements HasID<Integer>, Serializable {
    private Integer ID;
    private Integer zborID;
    private Integer clientID;
    private String turisti;
    private Integer locuri;

    public Bilet(Integer ID, Integer zborID, Integer clientID, String turisti, Integer locuri) {
        this.ID = ID;
        this.zborID = zborID;
        this.clientID = clientID;
        this.turisti = turisti;
        this.locuri = locuri;
    }

    @Override
    public Integer getID() {
        return this.ID;
    }

    @Override
    public void setID(Integer ID) {
        this.ID = ID;
    }

    public Integer getZborID() {
        return this.zborID;
    }

    public void setZborID(Integer zborID) {
        this.zborID = zborID;
    }

    public Integer getClientID() {
        return this.clientID;
    }

    public void setClientID(Integer clientID) {
        this.clientID = clientID;
    }

    public String getTuristi() {
        return turisti;
    }

    public void setTuristi(String turisti) {
        this.turisti = turisti;
    }

    public Integer getLocuri() {
        return locuri;
    }

    public void setLocuri(Integer locuri) {
        this.locuri = locuri;
    }


    @Override
    public String toString() {
        return "Bilet{" +
                "ID=" + ID +
                ", zborID=" + zborID +
                ", clientID=" + clientID +
                ", turisti='" + turisti + '\'' +
                ", locuri=" + locuri +
                '}';
    }
}
